### ISSUE TITLE
Closed menu items can still receive focus

### ISSUE THREAD

**Issue by TrebBrennan** [govau/designsystem#67]
RE: #54 

# What happened
On mobile, when the main-menu is closed (items are hidden) the menu items can still receive focus.

# Expected
Focus skips over hidden menu items.

# Notes
- Seems to be caused by `height: 0` on `.js .au-accordion__body.au-accordion--closed` Should be `display:none`
- See also #80
